tscached/handler_meta.py

```diff
@@ -37,10 +38,15 @@ def metadata_caching(config, name, endpoint, post_data=None):
     else:
         logging.info('Meta Endpoint MISS: %s' % redis_key)
         url = 'http://%s:%s%s' % (config['kairosdb']['host'], config['kairosdb']['port'], endpoint)
-        if post_data:
-            kairos_result = requests.post(url, data=post_data)
-        else:
-            kairos_result = requests.get(url)
+
+        try:
+            if post_data:
+                kairos_result = requests.post(url, data=post_data)
+            else:
+                kairos_result = requests.get(url)
+        except requests.exceptions.RequestException as e:
```

Comment:
Is a "requests.exceptions.RequestException" only raised on a failure to connect to the URI? I figure that's the case, but just wanted to make sure since you process bad responses (i.e. non 2XX) below.